app/resources/en/translation/executors/dealingwithestate.json

```diff
@@ -11,7 +11,12 @@
       "required": {
         "summary": "Select at least one executor",
         "message": "Select at least one executor"
-      }
+      },
+      "invalid":{
+        "summary": "A maximum of 4 executors can named on the grant of probate. ",
```

Comment:
I think we need to add 'be' in between 'can' and 'named.

Reply:
yes, good spot